shared/src/main/scala/cli/CLI.scala

```diff
@@ -39,6 +39,16 @@ object CLI:
     OParser.parse(parser, args, CLIConfig()) match
       case Some(config) =>
         val inputList = config.inputs.reverse.map(Parser.parseInput)
+        val shortDict =
+          io.Source.fromFile("resources/ShortDictionary.txt").getLines()
+        val longDict =
+          io.Source.fromFile("resources/LongDictionary.txt").getLines()
```

Comment:
```suggestion
        val shortDict =
          io.Source.fromResource("ShortDictionary.txt").getLines()
        val longDict =
          io.Source.fromResource("LongDictionary.txt").getLines()
```
Files in the `resource/` directory are sorta special, they're bundled into JARs (or native executables, when using Scala Native). Hopefully we can do something like this for Scala.js too